.github/workflows/tests-yaml-lint.yml

```diff
@@ -0,0 +1,11 @@
+name: Yaml Lint
+on: [push, pull_request]
+jobs:
+  lintAllTheThings:
```

Comment:
Personal opinion here - but prefer job to be simply called yamllint

Reply:
Changed.